### PR TITLE
Update credits for 5.7.0

### PR DESCRIPTION
**Pull request by sfan5**

### CODE REVIEW

File: builtin/mainmenu/tab_about.lua [hunk 1]
```diff
@@ -40,21 +40,17 @@ local core_team = {
...
 	"Lexi Hale [Particlespawner animation]",
-	"Liso [Shadow Mapping]",
-	"JosiahWI [Fixes, build system]",
-	"numzero [Graphics and rendering]",
-	"HybridDog [Fixes]",
-	"NeroBurner [Joystick]",
-	"pecksin [Clickable web links]",
+	"savilli [Bugfixes]",
+	"fluxionary [Bugfixes]",
+	"Gregor Parzefall [Documentation]",
```
Comment (grorp) [inline builtin/mainmenu/tab_about.lua:50]: Can `[Documentation]` be changed into `[Bugfixes]` or something? That's what I did the most according to https://github.com/minetest/minetest/pulls?q=is%3Apr+author%3Agrorp+is%3Amerged . I made very few changes changes to the documentation.

**Review by rubenwardy — APPROVED** [review]

File: builtin/mainmenu/tab_about.lua [hunk 2]
```diff
@@ -124,6 +128,12 @@ return {
 
 		local credit_list = {}
 		table.insert_all(credit_list, {
+			core.colorize("#222", "Dedication of the current release"),
```
Comment (Desour) [inline builtin/mainmenu/tab_about.lua:131]: The color is a little hard to read.

Comment (Desour) [reply builtin/mainmenu/tab_about.lua:131]: Maybe an orange color (for better emphasis and less brightness than yellow):
```suggestion
			core.colorize("#fc0", "Dedication of the current release"),
```

**Review by nerzhul — APPROVED** [review]

File: builtin/mainmenu/tab_about.lua (hunk 2 above)
Comment (sfan5) [reply builtin/mainmenu/tab_about.lua:131]: Agreed on the readability but the thought was to have a less happy color.

**Pull request merged**